.circleci/workflows.yml

```diff
@@ -180,19 +195,19 @@ jobs:
           restore-from-job: x86_64-linux-build
 
   x86_64-linux-dist-targets:
-    executor: docker-ubuntu-20
+    executor: docker-x86_64-ubuntu-20
     resource_class: large # 4-core
     environment:
       FERROCENE_HOST: x86_64-unknown-linux-gnu
-      FERROCENE_TARGETS: << pipeline.parameters.targets--x86_64-unknown-linux-gnu--std-only >>
+      FERROCENE_TARGETS: << pipeline.parameters.targets--x86_64-unknown-linux-gnu--std >>
```

Comment:
Are `*--build` the cross compilation targets and `*--std` the host targets?

Reply:
This would be a good rule of thumb, yes!